src/proj.h

```diff
@@ -287,14 +287,13 @@ PJ_OBS   proj_trans_obs   (PJ *P, enum proj_direction direction, PJ_OBS obs);
 PJ_COORD proj_trans_coord (PJ *P, enum proj_direction direction, PJ_COORD coord);
 
 
-int proj_transform (
+size_t proj_transform (
```

Comment:
I suggested to get rid of most, if not all, include files at top of proj.h, but the use of size_t requires #include <stddef.h>.